kura/org.eclipse.kura.linux.net/META-INF/MANIFEST.MF

```diff
@@ -12,6 +12,7 @@ Import-Package: javax.microedition.io,
  org.apache.commons.io;version="2.4.0",
  org.eclipse.kura;version="[1.1,2.0)",
  org.eclipse.kura.comm;version="[1.0,2.0)",
+ org.eclipse.kura.configuration;version="[1.0,1.1)",
```

Comment:
@ibinshtok If I'm not wrong, this should be [1.0,2.0) since you are using Password and not implementing it. Or you are using configuration to implement a configurable component?